src/DAL/repositories/partialTaskRepository.ts

```diff
@@ -45,9 +61,17 @@ export class PartialTaskRepository extends Repository<PartialTaskEntity> {
   }
 
   public async deletePartialTask(task: IPartialTaskParams): Promise<DeleteResult> {
-    // this.appLogger.log('info', `Deleting partial tasks for discrete "${discrete.id}" with version "${discrete.version}"`);
+    this.appLogger.log('info', `Deleting partial task with id "${task.id}"`);
```

Comment:
add discreate details as task id don't have useful info on its on (if it isn't in the DB)